staging/src/k8s.io/apiserver/pkg/util/webhook/client.go

```diff
@@ -228,3 +231,21 @@ func (cm *ClientManager) HookClient(cc ClientConfig) (*rest.RESTClient, error) {
 
 	return complete(cfg)
 }
+
+func isLocalHost(URL string) bool {
+	u, err := url.Parse(URL)
+	if err != nil {
+		return false
+	}
+
+	host := u.Hostname()
+	if host == "localhost" {
```

Comment:
does case matter here?

Reply:
domain name is case insensitive so I should fix it.